### ISSUE TITLE
Settings UX: some elements not read out properly

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52582]
**Environment Details:**
VSCode Version : 1.24.1
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 1.3.1
Screen Reader Versions : NVDA, JAWS18.0.5038

**Repro Steps:**
1.Launch VS Code while NVDA/JAWS is ON.
2.Navigate to Settings icon in Activity Bar->Press Enter->Select "Settings" from dropdown-->New settings editor-->Show Modified only checkbox and mark it as check-->Workbench-->Combo box present above "Reset"
3.Navigate to the combo box displayed.

**Actual:**
NVDA does not read the name of the combo box /also it does not read the state of the control present , while it reads the name of all the list items or elements present inside the combo box.
Also, NVDA does not read anything for "Reset" control present below the combo box.

**Expected:**
NVDA should read the name of the combo box/also it should read the state of the control present ,  while it reads the name of all the list items or elements present inside the combo box.
Also, NVDA should read the visually available name  for the "Reset" control present below the combo box.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The user will not be able to get clarity about control's name,state,role,value and other important info if only "TAB" is read by the screen reader

**MAS Reference:**
MAS 1.3.1 - https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment For Reference:**
[[Accessibility]A11y_VSCode_PreviewSettings_ScreenReader.pptx](https://github.com/Microsoft/vscode/files/2123164/Accessibility.A11y_VSCode_PreviewSettings_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by roblourens** [comment]
The new settings editor is a "preview" and isn't completely accessible.

**Comment by isidorn** [comment]
@roblourens there are around 5 accessibility issues around settings UX. It seems like there are not duplicates, just we should keep them in mind when we decide to take the settings editor out of preview. Leaving it up to you

**Comment by roblourens** [comment]
This was fixed at the end of august
